crates/grafbase-sdk/src/extension/authentication.rs

```diff
@@ -110,6 +117,10 @@ pub fn register<T: AuthenticationExtension>() {
         fn authenticate(&mut self, headers: &GatewayHeaders) -> Result<Token, ErrorResponse> {
             AuthenticationExtension::authenticate(&mut self.0, headers)
         }
+
+        fn public_metadata(&mut self) -> Result<Vec<crate::wit::PublicMetadataEndpoint>, Error> {
```

Comment:
Is it fine to use the record generated from WIT directly for `PublicMetadataEndpoint`, or would you have a conversion to a manually implemented struct?